pkgs/by-name/ay/ayatana-indicator-messages/package.nix

```
@@ -40,6 +40,10 @@ stdenv.mkDerivation (finalAttrs: {
     "dev"
   ] ++ lib.optionals withDocumentation [ "devdoc" ];
 
+  patches = [
+    ./fix-pie.patch
```

Just to track where this is from, why we're not `fetchpatch`ing it, and when this can be removed:

```suggestion
    # Remove when https://github.com/AyatanaIndicators/ayatana-indicator-messages/pull/39 merged & in release
    ./fix-pie.patch
```

---

Done. By the way, we did not get around to enabling PIE this staging round. Maybe next time..